cloud/amazon/ec2_customer_gateway.py

```diff
@@ -208,19 +217,22 @@ def main():
     name = module.params.get('name')
 
     existing = gw_mgr.describe_gateways(module.params['ip_address'])
+    # describe_gateways returns a key of CustomerGateways where as create_gateway returns a
+    # key of CustomerGateway. For consistency, change it here
+    existing['CustomerGateway'] = existing['CustomerGateways']
```

Comment:
I think this should be `existing['CustomerGateway'] = existing['CustomerGateways'][0]`because Gateways is an array, not an object.